File: Python/Program/Crops.py
class Crops:
    def __init__(self, name: str, type_: str, season: str):
        self._name = name
        self._type = type_
        self._season = season

    def get_name(self) -> str:
        return self._name

    def set_name(self, name: str):
        self._name = name

    def get_type(self) -> str:
        return self._type

    def set_type(self, type_: str):
        self._type = type_

    def get_season(self) -> str:
        return self._season

    def set_season(self, season: str):
        self._season = season

File: C++/Program/Crops.cpp
#pragma once
#include <iostream>
#include <string>

using namespace std;

class Crops {
private:
    string name;
    string type;
    string season;

public:
    // Constructor
    Crops(){}
    
    Crops(string name, string type, string season) {
        this->name = name;
        this->type = type;
        this->season = season;
    }

    // Getter and Setter for name
    string getName() {
        return name;
    }

    void setName(string name) {
        this->name = name;
    }

    // Getter and Setter for type
    string getType() {
        return type;
    }

    void setType(string type) {
        this->type = type;
    }

    // Getter and Setter for season
    string getSeason() {
        return season;
    }

    void setSeason(string season) {
        this->season = season;
    }
};